assets/js/driftRadiator.js

```diff
@@ -0,0 +1,77 @@
+dayjs.extend(window.dayjs_plugin_relativeTime)
+
+const lastIds = Object.fromEntries(environments.map(e => [`v:${e}`, 0]))
+const data = Object.fromEntries(environments.map(e => [`${e}`, undefined]))
+
+const tileLine = (label, value) => `<strong>${label}:</strong><span class="tile-value">${value}</span><br/>`
+const tileLink = (title, href) => `<a class="govuk-link--no-visited-state" href="${href}">${title}</a>`
+
+const updateVersion = ({ env, version }) => {
+  const [date, build, sha] = version.split('.')
+  data[env] = { version, date, build, sha }
+
+  $(`#${env}_details`).html(`
+    ${tileLine('Environment', env)}
+    ${tileLine('Build date', date)}
+    ${tileLine('Build number', build)}`)
+
+  if (env != devEnvName && data[devEnvName]) {
+    const devEnvSha = data[devEnvName].sha
+    const thisEnvSha = data[env].sha
+
+    const devEnvDate = data[devEnvName].date
+    const thisEnvDate = data[env].date
+
+    const humanReadableDate = `${dayjs(devEnvDate).diff(thisEnvDate, 'day')} days behind dev`
+    const gitDiffUrl = `https://github.com/ministryofjustice/${componentName}/compare/${thisEnvSha}...${devEnvSha}`
```

Comment:
This allows you to see the difference in commits between this env and dev in github, e.g: [link](https://github.com/ministryofjustice/create-and-vary-a-licence-api/compare/f35bf67...fc8a524)

This provides one of the benefits of the deployment tool - allows you to work out what tickets are inflight.
(The other bit of the deployment tool is pulling out jira ticket ids and displaying their current ticket state)